On branch main
Your branch is up to date with 'origin/main'.

Changes not staged for commit:
  (use "git add <file>..." to update what will be committed)
  (use "git restore <file>..." to discard changes in working directory)
	modified:   ssd/modeling/backbones/basic.py

no changes added to commit (use "git add" and/or "git commit -a")
fc8dff9707ebd557422cbb26302646531fc51511
diff --git a/assignment4.1/SSD/ssd/modeling/backbones/basic.py b/assignment4.1/SSD/ssd/modeling/backbones/basic.py
index 91ef094..0168be2 100644
--- a/assignment4.1/SSD/ssd/modeling/backbones/basic.py
+++ b/assignment4.1/SSD/ssd/modeling/backbones/basic.py
@@ -1,3 +1,4 @@
+import torch
 from torch import nn
 from typing import Tuple, List
 
